ui-mockups/src/articles.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]><!-->
+<html class="no-js">
+<!--<![endif]-->
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title></title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Vidaloka" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU"
+        crossorigin="anonymous">
+    <link rel="stylesheet" href="css/bootstrap.css">
+    <link rel="stylesheet" href="css/style.css">
+
+</head>
+
+<body>
+    <nav class="navbar navbar-expand-lg navbar-light">
+        <div class="container">
+            <div></div>
+            <a class="navbar-brand font-weight-bold" href="#">Authors' Haven</a>
+            <div></div>
+        </div>
+    </nav>
+    <div class="container">
+        <div class="row">
+            <div class="col-sm-4">
+                <div class="card">
+                    <img class="card-img-top" src="assets/alisa-anton-632369-unsplash.jpg" alt="Card image cap">
+                    <div class="card-body">
+                        <h5 class="card-title">Card title</h5>
+                        <p class="card-text ">
+                            Some quick example text to build on the card title and make up the bulk of
+                            the card's content.
+                        </p>
+                        <small>
+                            <a class="ah-author-link" href="#">Meshack Mbuvi</a>
+                        </small>
+                        <p class="card-text d-flex justify-content-between">
+                            <small class="text-muted">July 11.</small>
+                            <small class="text-muted">7 min read</small>
+                        </p>
+                    </div>
+                </div>
+            </div>
+            <div class="col-sm-4">
+                <div class="list-group">
+                    <div class=" d-flex justify-content-between ">
+                        <img src="assets/annie-spratt-558900-unsplash.jpg" class="rounded float-left" alt="post image"
+                            width="120" height="120">
+                        <div class="ml-2">
+                            <h6><a href="article.html">Donec id elit non mi porta</a></h6>
```

Comment:
For the article title, remove the blue color on hover. Have it black with underline